src/Access/MediaAvailableAccess.php

```diff
@@ -206,7 +206,8 @@ protected function getStubMediaObject(MediaInterface $media): MpxMedia {
    */
   private function getDateTime(MediaInterface $media, string $field_name): DateTimeFormatInterface {
     $fieldItemList = $media->get($field_name);
-    $date = !$fieldItemList->isEmpty() ? new ConcreteDateTime(\DateTime::createFromFormat(DateTimeItemInterface::DATETIME_STORAGE_FORMAT, $fieldItemList->value)) : new NullDateTime();
+    $date_time = \DateTime::createFromFormat(DateTimeItemInterface::DATETIME_STORAGE_FORMAT, $fieldItemList->value);
```

Comment:
What is $fieldItemList->value when it fails? An empty string? I'm surprised the isEmpty() check isn't covering this.

Reply:
I debugged and got an integer value which is a valid timestamp value